extensions/PhabBugz/lib/Constants.pm

```diff
@@ -32,4 +37,22 @@ use constant PHAB_USER_POLL_SECONDS  => $ENV{PHAB_USER_POLL} // 60;
 use constant PHAB_GROUP_POLL_SECONDS => $ENV{PHAB_GROUP_POLL} // 300;
 use constant PHAB_TIMEOUT            => $ENV{PHAB_TIMEOUT} // 60;
 
+use constant LANDO_AUTOMATION_USER   => 'lobot@bmo.tld';
+use constant LANDO_BUG_UPDATE_FIELDS => qw(
+  cf_status_firefox
```

Comment:
When we want to add new fields to the Lando endpoint we can just add them here, right?? That would be quite convenient.

Reply:
> When we want to add new fields to the Lando endpoint we can just add them here, right?? That would be quite convenient.

That is the goal, yes.